drakrun/drakrun/run.py

```diff
@@ -35,3 +36,5 @@ def run_vm(vm_id):
 
         with open("/var/log/xen/qemu-dm-vm-{vm_id}.log".format(vm_id=vm_id), "rb") as f:
             logging.error(f.read())
+
+    return snapshot_version
```

Comment:
idk if this is sane idea to return it from `run_vm` but at least it was simple :thinking: